src/RpcServer/RpcServer.SmartContract.cs

```diff
@@ -92,5 +96,27 @@ private JObject InvokeScript(JArray _params)
             CheckWitnessHashes checkWitnessHashes = _params.Count >= 2 ? new CheckWitnessHashes(((JArray)_params[1]).Select(u => UInt160.Parse(u.AsString())).ToArray()) : null;
             return GetInvokeResult(script, checkWitnessHashes);
         }
+
+        [RpcMethod]
+        private JObject GetUnclaimedGas(JArray _params)
+        {
+            string address = _params[0].AsString();
+            JObject json = new JObject();
+            UInt160 script_hash;
+            try
+            {
+                script_hash = address.ToScriptHash();
+            }
+            catch
+            {
+                script_hash = null;
+            }
+            if (script_hash == null)
+                throw new RpcException(-100, "Invalid address");
+            SnapshotView snapshot = Blockchain.Singleton.GetSnapshot();
+            json["unclaimed"] = NativeContract.NEO.UnclaimedGas(snapshot, script_hash, snapshot.Height + 1).ToString();
```

Comment:
why +1?

Reply:
I did not think about it, I just copied from here https://github.com/neo-project/neo-modules/blob/82147d5592ada1ad7fd0d2888762963800f8c0d7/src/RpcServer/RpcServer.Wallet.cs#L78

Reply:
Maybe because your "claim TX" always will enter in the next block, hahahah I don't know :P

Reply:
Do you think we need to change it now?

Reply:
it's ok for me